lib/govuk_app_config/configure.rb

```diff
@@ -3,21 +3,11 @@
 end
 
 GovukError.configure do |config|
-  # We're misusing the `should_capture` block here to hook into raven until
-  # there's a better way: https://github.com/getsentry/raven-ruby/pull/750
-  config.should_capture = Proc.new { |e|
+  config.before_send = Proc.new { |e|
     GovukStatsd.increment("errors_occurred")
 
     # For backwards compatibility
     GovukStatsd.increment("errbit.errors_occurred")
-
-    exception_class = e.respond_to?(:original_exception) ? e.original_exception.class : e.class
-    if exception_class.ancestors.any? { |c| c.name =~ /^GdsApi::(HTTPIntermittent|TimedOutException)/ }
-      GovukStatsd.increment("gds_api_adapters.errors.#{e.class.name.demodulize.underscore}")
```

Comment:
I wonder if there's any way we can keep this behaviour? It was useful being able to see graphs like this:

![download](https://user-images.githubusercontent.com/510498/51591517-1a5a9980-1ee5-11e9-9837-927511263460.png)

Perhaps we could even make it generic and just record stats for every kind of error?

Reply:
Would `GovukStatsd.increment("errors.#{e.class.name.demodulize.underscore}")` work? It would be generic but not backwards compatible.

Reply:
I think that sounds good - we can always combine the old and the new data in graphs if you need both.

Reply:
+1 for this - on Content Publisher we're trying to track how a big a problem the HTTP intermittents are for us